### PR TITLE
Update node and npm versions in README [2.0]

### PR DESCRIPTION
**Pull request by maya**
Update node and npm versions to what we currently use in the README. 
Semi-related to #2954.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,15 +129,15 @@ And that’s it — you should now be able to copy our code samples into your `i
 
 1. Install `Node/npm`. Below is a link to find the install method that coincides with your operating system:
 
-    - Node v4.2.3+, [Installation guides](https://nodejs.org/en/download/)
+    - Node v10.15.1+, [Installation guides](https://nodejs.org/en/download/)
```
Comment (maya) [inline README.md]: Do we know if anything above 10 works? e.g. does node 11 work?

Comment (thisisdano) [reply README.md]: Untested afaik

Comment (maya) [reply README.md]: Updated to be explicitly node 10 in f65e9fb.

**Comment by saracope** [comment]
The circleci config is still using a node 8 image so we should bump that too. @maya  Do you want to make that change in the PR or do is separately?

**Comment by maya** [comment]
@saracope We can do it in a separate PR.

**Comment by maya** [comment]
Also, what's the proper way to do that? Right now, it's `circleci/node:8.10-browsers` does it need to change to `circleci/node:10.15.1-browsers`?

**Comment by saracope** [comment]
Yeah

**Pull request merged**